<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebClientDemo</class>
 <widget class="QWidget" name="WebClientDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WebClientDemo</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect to Server URL:</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSendMsg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Send  Msg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConnect">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>521</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editURL">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
    </font>
   </property>
   <property name="text">
    <string>ws://localhost:1234</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editSendMsg">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhMultiLine</set>
   </property>
   <property name="text">
    <string>{     sendToPlugin({         'from_property' : 'connected'     }); }</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>137</y>
     <width>256</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCalifornia">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Manual</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnSendMsg</sender>
   <signal>clicked()</signal>
   <receiver>WebClientDemo</receiver>
   <slot>btnSendMsg_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>WebClientDemo</receiver>
   <slot>btnClose_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnConnect</sender>
   <signal>clicked()</signal>
   <receiver>WebClientDemo</receiver>
   <slot>btnConnect_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCalifornia</sender>
   <signal>clicked(bool)</signal>
   <receiver>WebClientDemo</receiver>
   <slot>btnManual_clicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>605</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btnSendMsg_clicked()</slot>
  <slot>btnClose_clicked()</slot>
  <slot>btnConnect_clicked()</slot>
  <slot>btnManual_clicked(bool)</slot>
 </slots>
</ui>
